src/test/java/duke/ParseTest.java

```diff
@@ -0,0 +1,15 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+public class ParseTest {
+    @Test
+    public void deadlineDetails_success() {
+        Parser parser = new Parser();
+        String deadline = "homework /by: 2023-12-01T12:20:00";
+        String[] output = parser.deadlineDetails(deadline);
```

Comment:
To get the deadline details, renaming the method to getDeadlineDetails would fit the naming conventions better